loader/src/platform/windows/main.cpp

```diff
@@ -138,11 +138,29 @@ void* mainTrampolineAddr;
 #include "gdTimestampMap.hpp"
 unsigned int gdTimestamp = 0;
 
+// In case the game is launched from a different directory through command line
+// this function will set the current working directory to the game's directory
+// to avoid the game crashing due to not being able to find the resources
+static void fixCWD() {
+    char cwd[1024];
```

Comment:
MAX_PATH should be used instead of an arbitrary number

Reply:
I was using a different number since i found max path to be rather small but sure